olp-cpp-sdk-core/src/http/curl/NetworkCurl.cpp

```diff
@@ -579,21 +595,22 @@ ErrorCode NetworkCurl::SendImplementation(
 }
 
 void NetworkCurl::Cancel(RequestId id) {
-  OLP_SDK_LOG_TRACE(kLogTag, "cancel with id = " << id);
-
   if (!IsStarted()) {
+    OLP_SDK_LOG_ERROR(kLogTag, "Can't cancel the request id="
```

Comment:
Why is this an error? I would say missing accomplished, silent exit.

Reply:
We can't continue the operation, thus it looks more as an error.